### DISCUSSION TITLE
Wave Table with collapsed rows

### DISCUSSION THREAD

**Discussion by n-srinidhi** [h2oai/wave#1957] [Q&A] [answered]
Hi!I'm trying to build a Wave Table with collapsed rows. I would like to achieve the following display:

![image](https://user-images.githubusercontent.com/117831190/235618798-f138d82e-efcb-4aad-9914-70e244ee4a86.png)

Basically a table with collapsed groups and 2 columns for each group.

And upon expansion of the group,
![image](https://user-images.githubusercontent.com/117831190/235619047-21e71e81-e237-4cd9-85ff-7e881e14415a.png)

Is it possible to achieve this in Wave?I need to show the summed up value of the 'Value' column for each group.

**Comment by mturoci** [comment]
Looking for [this](https://wave.h2o.ai/docs/widgets/form/table/#with-collapsed-groups)?

**Reply by n-srinidhi** [reply]
@mturoci Thanks for the prompt response!Doesn't the example linked above only allow a single title?
Like I want to display both User ID plus the Value in place of assigned to bob.

**Reply by mturoci** [reply, marked as answer]
> I want to display both User ID plus the Value in place of assigned to bob.

I see. In that case, your best option is to compute the total value yourself and make it part of the title. Something like

```py
total_val = 10
ui.table_group(f"Assigned to Bob {total_val}", [
    ui.table_row(name='row1', cells=['Task1', 'Moderate']),
    ui.table_row(name='row2', cells=['Task2', 'High'])
], collapsed=False),
```

Wave doesn't do this automatically since it may not suit everyone.

**Reply by n-srinidhi** [reply]
@mturoci Thank you!
